eng/service.proj

```diff
@@ -11,17 +11,20 @@
     <IncludeStress Condition="'$(IncludeStress)' == ''">true</IncludeStress>
     <IncludeSamplesApplications Condition="'$(IncludeSamplesApplications)' == ''">true</IncludeSamplesApplications>
     <IncludeSamplesApplications Condition="'$(ServiceDirectory)' != '*' or '$(IncludeSamples)' == 'false'">false</IncludeSamplesApplications>
+    <IsClientLibrary Condition="'$(IsClientLibrary)' == ''">true</IsClientLibrary>
   </PropertyGroup>
 
   <ItemGroup>
     <MgmtExcludePaths Include="$(MSBuildThisFileDirectory)..\sdk\*\Microsoft.*.Management.*\**\*.csproj;$(MSBuildThisFileDirectory)..\sdk\*mgmt*\**\*.csproj" />
     <TestProjects Include="..\sdk\$(ServiceDirectory)\$(Project)\tests\**\*.csproj" />
+    <ClientTestProjects Include="..\sdk\$(ServiceDirectory)\$(Project)\tests\**\Azure.*csproj" />
```

Comment:
We want to try and avoid this globbing because we want to support client libraries that don't start with Azure as well.